js/feedback-form.js

```diff
@@ -1,49 +1,51 @@
-jQuery(document).ready(function($) {
-	var display_error, hide_error,
-		$textarea_feedback = $( '#feedback-text' ),
-		$first_screen = $( '.draftfeedback-first-screen' ),
-		$second_screen = $( '.draftfeedback-second-screen' );
-
-	display_error = function( id, notice, hide_on_keydown ) {
-		var error_container = $( '#draft-error' );
-		if ( error_container.length) {
-			hide_error( false, true ).done( function() {
-				display_error( id, notice, hide_on_keydown );
+jQuery( document ).ready(function( $ ) {
+	var displayError, hideError,
+		$textareaFeedback = $( '#feedback-text' ),
+		$firstScreen = $( '.draftfeedback-first-screen' ),
+		$secondScreen = $( '.draftfeedback-second-screen' );
+
+	displayError = function( id, notice, hideOnKeydown ) {
+		var errorContainer = $( '#draft-error' );
+		if ( errorContainer.length ) {
+			hideError( false, true ).done( function() {
+				displayError( id, notice, hideOnKeydown );
 			});
 			return;
 		} else {
-			error_container = '<div id="draft-error" class="error"><p>' + notice + '</p></div>';
+			errorContainer = '<div id="draft-error" class="error"><p>' + notice + '</p></div>';
 		}
 
-		$(id).after(error_container);
-		if ( hide_on_keydown ) {
-			$( hide_on_keydown ).on( 'keydown', function() {
-				hide_error( false );
+		$( id ).after( errorContainer );
+		if ( hideOnKeydown ) {
+			$( hideOnKeydown ).on( 'keydown', function() {
+				hideError( false );
 			});
 		} else {
-			hide_error( true );
+			hideError( true );
 		}
-	}
+	};
 
-	hide_error = function( delayed, fast ) {
-		var error_container = $('#draft-error');
+	hideError = function( delayed, fast ) {
+		var errorContainer = $( '#draft-error' );
 
 		if ( delayed ) {
-			error_container = error_container.delay( 4000 );
+			errorContainer = errorContainer.delay( 4000 );
 		}
 
-		return error_container
+		return errorContainer
 			.fadeOut( fast ? 'fast' : 'slow' )
 				.promise()
-			.done( function() { $( '#draft-error' ).remove(); } );
+			.done( function() {
+ $( '#draft-error' ).remove();
```

Comment:
This change doesn't make sense to me, why not keep it all in one line? Also, indentation seems off.